fintech-examples/fintech-ui/src/app/services/storage.service.ts

```diff
@@ -61,7 +62,7 @@ export class StorageService {
       return null;
     }
     const date =  new Date(parseInt(validUntilTimestamp));
-    if (date.toLocaleString() === "Invalid Date") {
+    if (Consts.toLocaleString(date) === "Invalid Date") {
```

Comment:
Date validity can be checked via:
```
if(isNaN(date)) {
// handle invalid date
}
```

Reply:
This is better. Thx.

Reply:
const date = new Date(parseInt(validUntilTimestamp));
    if (isNaN(date)) {
      return null;
    }
**Argument of type date is not assignable to parameter of type number.** 

The weired thing is, that sometimes I got a number, though the localStorage field was empty. And that resulted in a Invalid Date. Unfortunatly I can not reproduce error.